testing/test_aux.sh

```diff
@@ -63,11 +63,11 @@ site_path: inst/test_site
 schema_path: schemas/udmi
 interfaces:
   faux-1:
-    opts: brute broadcast_client ntpv4
+    opts: brute broadcast_client ntpv4 arp
```

Comment:
why is this enabled for everything?  Whatever it's for, that's not the right way... If it's truly always required then it should be the default behavior, otherwise all three of these should not have it (what's the point?)

Reply:
Ah, that would have been a much better, thanks. This is related to the min_send test which always passes, but might have a different summary depending if there are any ARP packets. As there seems to be some chance it might not, this periodically sends ARP packets so the result is always as expected.